_episodes/05-anaconda.md

```diff
@@ -29,6 +29,13 @@ Anaconda has easy to use installers for Windows, Mac, and Linux.  It installs a
 2. Download the installer. **Be sure you get the Python 3 version.**
 3. Double click the installer icon and follow the setup instructions.  You can use all of the default options.
 
+#### Linux - [Video Tutorial](https://www.youtube.com/watch?v=DY0DB_NwEu0)
+
+1. Open the [Anaconda Linux download page](https://www.anaconda.com/download/#linux).
+2. Download the installer using `wget` or `curl` downloader. **Be sure you get the Python 3 version.**
```

Comment:
They do no need to use `wget` or `curl` if they download the `sh` file directly from the URL. Clicking the 'Download' button should download an installer `sh` file which is then executed using points 3 and 4.

I do not see the link 'How to Install Anaconda' at the bottom of the page as referenced in the video, but I am on a Mac. 

Please verify these instructions by performing an installation.